### PR TITLE
[IMP] website, ecommerce: new amazing dashboard #somuchwow

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Task: https://www.odoo.com/web?debug=#id=29247&view_type=form&model=project.task&menu_id=5200

### CODE REVIEW

File: addons/sale/__manifest__.py [hunk 1]
```diff
@@ -3,49 +3,16 @@
 
 {
     'name': 'Sales',
-    'version': '1.0',
+    'version': '1.1',
     'category': 'Sales',
-    'sequence': 15,
```
Comment (nhomar) [inline addons/sale/__manifest__.py]: Just to understand, Why remove the sequence?

File: addons/sale/__manifest__.py [hunk 2]
```diff
@@ -68,5 +35,4 @@
     'css': ['static/src/css/sale.css'],
     'installable': True,
     'auto_install': False,
-    'application': True,
```
Comment (nhomar) [inline addons/sale/__manifest__.py]: Why it is not an application anymore?

File: addons/sale/tests/__init__.py [hunk 3]
```diff
@@ -3,4 +3,3 @@
 import test_sale_to_invoice
 import test_sale_order
 from . import test_product_id_change
-import test_sale_ui
```
Comment (nhomar) [inline addons/sale/tests/__init__.py]: No more tests in a refactor?

Comment (tde-banana-odoo) [inline addons/sale/tests/__init__.py]: Those are moved in sales_management. Actually sales module will contain the logic, sales_management will be the app enabling its use. The purpose is that when you use the ecommerce, you don't have Sales + Invoice + Website apps, only Invoice and Website. You may activate sales feature by installing the sales management app that basically enables the display of various menus. In short: nothing changes, the purpose is to have less apps displayed in some specific use (i.e. i use only ecommerce, I use only Sales management, ...) .

**Pull request merged**